src/Diagnostics.DataProviders/ConfigurationFactory.cs

```diff
@@ -174,18 +175,76 @@ private void LoadConfigurationValues(object dataProviderConfiguration)
                         continue;
                     }
 
-                    var value = GetValue(prefix, attribute.Name);
+                    dynamic value;
+                    string valueType;
+                  
+                    if (property.Name == "OverridableExceptions")
+                    {
+                        List<ITuple> overridableExceptionsList = new List<ITuple>();
+                        for (int i = 0; i < int.Parse(GetValue(prefix, "Retry:OverridableExceptionsToRetryAgainstLeaderCluster:AmountOfOverridableExceptions")); i++)
```

Comment:
Consider using int.TryParse instead of int.Parse

Reply:
Is it possible to make this work without the property AmountOfOverridableExceptions?

Reply:
Should I put the for loop inside of an if block that executes if the try parse is successful?

I wasn't able to find a way to count how many objects were in the list or just grab the list and iterate through it. I did see a list of all the key value pairs from the config file in watcher but I couldn't find a way to access it in the code. Do you know of any ways to do this?

Reply:
Following code snippet can help read an array from config.

https://github.com/Azure/Azure-AppServices-Diagnostics-KustoIngestor/blob/b43309976067f879c4e7d0f56e8b5af089b9c7cc/Kusto.Data.Ingestor/ExecuteCustomLogAgregatorsFunction.cs#L74